src/components/Calendar/EventModal.tsx

```diff
@@ -43,9 +50,13 @@ const EventModal: React.FC<{
             isOpen={true}
             onClose={() => setEventClick(null)}
         >
-            <p>Start: {startDate}</p>
-            <p>End: {endDate}</p>
-            {eventClick.description ? <p>Description: {eventClick.description}</p> : <></>}
+            <StyledParagraph>Start: {startDate}</StyledParagraph>
+            <StyledParagraph>End: {endDate}</StyledParagraph>
+            {eventClick.description ? (
```

Comment:
What do you think about doing conditional rendering with `eventClick.description && <StyledParagraph>...`?

Reply:
Refactored